src/renderer/components/LoginForm.tsx

```diff
@@ -467,10 +472,9 @@ export function ConfigureProgressDialog({
   }, [accountId])
 
   return (
-    <Dialog
-      isOpen={isOpen}
+    <DeltaDialogBase
```

Comment:
Every dialogue uses now `DeltaDialogBase` in the end which makes it hopefully more clear how they all relate to each other:

- `Dialog` from `blueprintjs` > `DeltaDialogBase` > Any other dialogue